modules/nf-core/proteus/readproteingroups/templates/proteus_readproteingroups.R

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env Rscript
+
+################################################
+################################################
+## Functions                                  ##
+################################################
+################################################
```

Comment:
any licence or author for this script?

Reply:
Hmmm, regarding author, where do I write that? Just at the beginning of the script? And regarding licence, I assumed that would just be the licence of the tool which is mentioned in the meta.yml, no?

Reply:
You wrote the script, you can choose whichever license you want.
I do like MIT personally.
here are 2 examples from the rnaseq pipeline:
https://github.com/nf-core/rnaseq/blob/master/bin/prepare-for-rsem.py
https://github.com/nf-core/rnaseq/blob/master/bin/gtf2bed

Reply:
Done, many thanks!